tests/integration/test_methodologies_johnny_decimal_all.py

```
@@ -0,0 +1,1392 @@
+    def test_from_string_id(self) -> None:
+        from file_organizer.methodologies.johnny_decimal.categories import JohnnyDecimalNumber
+
+        num = JohnnyDecimalNumber.from_string("11.01.001")
+        assert num.area == 11
+        assert num.category == 1
+        assert num.item_id == 1
+
+    def test_from_string_invalid_raises(self) -> None:
+        from file_organizer.methodologies.johnny_decimal.categories import JohnnyDecimalNumber
+
+        with pytest.raises(ValueError):
```

Ruff (0.15.9)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)